packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -160,6 +153,15 @@ export const LivenessCameraModule = (
     videoWidth && videoHeight ? videoWidth / videoHeight : 0
   );
 
+  // Only allow on mobile screen if it is no light challenge
+  const allowSelectableDevicesMobile =
+    !isMobileScreen || isFaceMovementChallenge;
```

Comment:
It's a bit hard to follow that `allowSelectableDevicesMobile` is `true` when `isMobileScreen` is `false`. Realize that it going to be hard for this not to be somewhat confusing but maybe we can update the approach to something like:
```ts
const hasMultipleDevices = !!selectableDevices?.length;
const allowDeviceSelection = (isStartView && hasMultipleDevices) && (!isMobileScreen || isFaceMovementChallenge)
```

Reply:
Think that makes sense, think this is much cleaner